pandas/core/frame.py

```diff
@@ -9919,6 +9919,8 @@ def apply(
         result_type: Literal["expand", "reduce", "broadcast"] | None = None,
         args=(),
         by_row: Literal[False, "compat"] = "compat",
+        engine: str = "python",
+        engine_kwargs: dict = {},
```

Comment:
I think we usually default this as `engine_kwargs: dict[str, bool] | None = None`